GNUmakefile

```diff
@@ -68,4 +68,4 @@ ifeq (,$(wildcard $(GOPATH)/src/$(WEBSITE_REPO)))
 endif
 	@$(MAKE) -C $(GOPATH)/src/$(WEBSITE_REPO) website-provider-test PROVIDER_PATH=$(shell pwd) PROVIDER_NAME=$(PKG_NAME)
 
-.PHONY: build install uninstall test testacc vet fmt fmtcheck errcheck lint tools test-compile website website-lint website-test
\ No newline at end of file
+.PHONY: build install uninstall test testacc vet fmt fmtcheck errcheck lint tools test-compile website website-lint website-test
```

Comment:
still showing a change in makefile

Reply:
reverted this change, but I notice this happens with all my PRs because vscode adds a newline whenever i open/save this file - it should be ok to add a newline to the makefile right?

Reply:
huh, I use vscode too, but I've never had this problem. ya should be fine otherwise.